pages/common/grep.md

```diff
@@ -2,35 +2,36 @@
 
 > Matches patterns in input text.
 > Supports simple patterns and regular expressions.
+> More information: <https://man7.org/linux/man-pages/man1/grep.1.html>.
 
 - Search for a pattern within a file:
 
 `grep {{search_pattern}} {{path/to/file}}`
 
-- Search for an exact string:
+- Search for an exact ([F]ixed) string:
 
 `grep -F {{exact_string}} {{path/to/file}}`
 
 - Search for a pattern [R]ecursively in the current directory, showing matching line [n]umbers, [I]gnoring non-text files:
 
-`grep -RIn {{search_pattern}} .`
+`grep -RIn {{search_pattern}} {{.}}`
 
-- Use extended regular expressions (supporting `?`, `+`, `{}`, `()` and `|`), in case-insensitive mode:
+- Use [E]xtended regular expressions (supporting `?`, `+`, `{}`, `()` and `|`), in case-[i]nsensitive mode:
 
 `grep -Ei {{search_pattern}} {{path/to/file}}`
 
 - Print 3 lines of [C]ontext around, [B]efore, or [A]fter each match:
 
-`grep -{{C|B|A}} 3 {{search_pattern}} {{path/to/file}}`
+`grep -{{C|B|A}} {{3}} {{search_pattern}} {{path/to/file}}`
 
-- Print file name with the corresponding line number for each match:
+- Print file name [H]eaders with the corresponding line [n]umber for each match:
 
 `grep -Hn {{search_pattern}} {{path/to/file}}`
 
-- Use the standard input instead of a file:
+- Search for lines matching a pattern, printing [o]nly the matched text:
 
-`cat {{path/to/file}} | grep {{search_pattern}}`
+`grep -o {{search_pattern}}`
```

Comment:
what about the `stdin`?

Reply:
Well, I know that's useful, but I decided it was more of a general shell concept, and that since `-o` was useful and **not** as obvious, it would be better to have it instead.

Reply:
If you think a different example would be better to replace, I'm all for doing that.

Reply:
but this examples an input like {{path/to/file}} or the stdin

Reply:
I'm not sure what you mean&mdash;will you please clarify?

Reply:
If I run `$ grep -o text` the grep program waits for an input from `stdin` because I didn't specify any input file or files.

Reply:
Ah I see, you think I should add an example for that?

Reply:
Oh oops!!! I see now!  I didn't specify a file in this example, so it's defaulting to stdin, that's what you're noticing?